opcua.cpp

```diff
@@ -250,3 +309,71 @@ void OPCUAServer::stop()
 		m_server->Stop();
 	}
 }
+
```

Comment:
Documentation header would help the reader

Reply:
In my copy of he code there is one.